### PR TITLE
Add common exceptions to the TRACEBACK docs.

### PR DESCRIPTION
**Pull request by IsaacG**

### CODE REVIEW

File: docs/TRACEBACKS.md [hunk 1]
```diff
@@ -54,6 +54,32 @@ ValueError: not enough values to unpack (expected 2, got 1)
 Working backwards from the bottom, we see that the call where the exception happened is on line 2 in `my_func`.
 We got there by calling `my_func` on line 5.
 
+## Common Exceptions
+
+Python 3.11 defines 67 [built-in exception classes][exception-hierarchy].
+Here is a brief overview of some of the more common exceptions and what they indicate.
+
+* **SyntaxError**: Python is unable to understand the code as the code has invalid syntax.
+  For example, there may be an open parenthesis without a matching closing parenthesis.
```
[the remark this answers is not in the document]
Comment (IsaacG) [reply docs/TRACEBACKS.md]: Done.

File: docs/TRACEBACKS.md [hunk 2]
```diff
@@ -54,6 +54,32 @@ ValueError: not enough values to unpack (expected 2, got 1)
...
+* **AssertionError**: An `assert` statement (see below) failed.
+* **AttributeError**: The code (or unit test!) tried to access the attribute of an object but that object has no attribute.
+  For example, a unit test excepts a `Robot` object to have a `direction` attribute but when it tried to access `robot.direction`, it does not exist.
+  This could also indicate a typo, such as using `"Hello".lowercase()` when the correct syntax is `"Hello".lower()`.
+  `"Hello".lowercase()` raises `AttributeError: 'str' object has no attribute 'lowercase'`.
```
[the remark this answers is not in the document]
Comment (IsaacG) [reply docs/TRACEBACKS.md]: Done.

File: docs/TRACEBACKS.md [hunk 3]
```diff
@@ -54,6 +54,32 @@ ValueError: not enough values to unpack (expected 2, got 1)
...
+* **ImportError**: The code tried to import something, but it wasn't there.
+  For example, a unit test does `from exercise import Thing` but the `exercise.py` file does not define a `Thing`.
```
[the remark this answers is not in the document]
Comment (IsaacG) [reply docs/TRACEBACKS.md]: Done.

File: docs/TRACEBACKS.md [hunk 4]
```diff
@@ -54,6 +54,32 @@ ValueError: not enough values to unpack (expected 2, got 1)
 Working backwards from the bottom, we see that the call where the exception happened is on line 2 in `my_func`.
 We got there by calling `my_func` on line 5.
 
+## Common Exceptions
+
+Python 3.11 defines 67 [built-in exception classes][exception-hierarchy].
```
[the remark this answers is not in the document]
Comment (IsaacG) [reply docs/TRACEBACKS.md]: Done.

File: docs/TRACEBACKS.md [hunk 5]
```diff
@@ -54,6 +54,32 @@ ValueError: not enough values to unpack (expected 2, got 1)
...
+Here is a brief overview of some of the more common exceptions and what they indicate.
+
+* **SyntaxError**: Python is unable to understand the code as the code has invalid syntax.
+  For example, there may be an open parenthesis without a matching closing parenthesis.
+* **AssertionError**: An `assert` statement (see below) failed.
+* **AttributeError**: The code (or unit test!) tried to access the attribute of an object but that object has no attribute.
+  For example, a unit test excepts a `Robot` object to have a `direction` attribute but when it tried to access `robot.direction`, it does not exist.
```
[the remark this answers is not in the document]
Comment (IsaacG) [reply docs/TRACEBACKS.md]: Done.

File: docs/TRACEBACKS.md [hunk 6]
```diff
@@ -54,6 +54,336 @@ ValueError: not enough values to unpack (expected 2, got 1)
...
+E   AssertionError
+```
+
+
+</details>
+
+
+### **AttributeError**
+An `AttributeError` is raised when code (or a unit test!) tries to access the attribute of an object but that object has no such attribute.
+For example, a unit test excepts a `Robot` object to have a `direction` attribute but when it tried to access `robot.direction`, it does not exist.
```
Comment (BethanyG) [inline docs/TRACEBACKS.md:143]:
```suggestion
For example, a unit test expects a `Robot` object to have a `direction` attribute, but when it tried to access `robot.direction`, it does not exist.
```

File: docs/TRACEBACKS.md [hunk 7]
```diff
@@ -54,6 +54,336 @@ ValueError: not enough values to unpack (expected 2, got 1)
...
+    return _bootstrap._gcd_import(name[level:], package, level)
+.mnt.exercism-iteration.lasagna_test.py:23: in <module>
+    raise ImportError("In your 'lasagna.py' file, we can not find or import the"
+
+E   ImportError: In your 'lasagna.py' file, we can not find or import the function named 'bake_time_remaining()'. Did you mis-name or forget to define it?
+```
+
+### **IndexError**
+
+Python raises an `IndexError` when an invalid index is used to look up a value in a list.
```
Comment (BethanyG) [inline docs/TRACEBACKS.md:245]:
```suggestion
Python raises an `IndexError` when an invalid index is used to look up a value in a sequence.
```

File: docs/TRACEBACKS.md [hunk 8]
```diff
@@ -54,6 +54,336 @@ ValueError: not enough values to unpack (expected 2, got 1)
...
+    return 'Hello, ' + name + '!'
+E   TypeError: can only concatenate str (not "int") to str
+```
+
+
+</details>
+
+
+### **ValueError**
+A `ValueError` is usually raised when an invalid value is passed to function.
```
Comment (BethanyG) [inline docs/TRACEBACKS.md:356]:
```suggestion
A `ValueError` is usually raised when an invalid value is passed to function.


```

File: docs/TRACEBACKS.md [hunk 9]
```diff
@@ -54,6 +54,336 @@ ValueError: not enough values to unpack (expected 2, got 1)
...
+<details>
+<summary>Click here for code example</summary>
+
+
+Note, real square roots only exist for position numbers.
```
Comment (BethanyG) [inline docs/TRACEBACKS.md:361]:
```suggestion
Note, real square roots only exist for positive numbers.
```

**Review by BethanyG — APPROVED** [review]
Just a few last things, otherwise its good to go!  **Many thanks** for this! 🌟

**Pull request merged**